addons/l10n_it_stock_ddt/data/l10n_it_ddt_template.xml

```diff
@@ -5,7 +5,7 @@
                 <t t-if="ddt_dict and not record.l10n_it_ddt_id">
                     <t t-foreach="ddt_dict" t-as="picking">
                         <DatiDDT>
-                            <NumeroDDT t-esc="format_alphanumeric(picking.l10n_it_ddt_number[-20:])"/>
+                            <NumeroDDT t-esc="format_alphanumeric(picking.l10n_it_ddt_number[-20:]) if picking.l10n_it_ddt_number else ''"/>
```

Comment:
```suggestion
                            <NumeroDDT t-if="picking.l10n_it_ddt_number" t-esc="format_alphanumeric(picking.l10n_it_ddt_number[-20:])"/>
```
Do we want the `NumeroDDT` to be displayed at all if there is no value ? Isn't this more correct ? :thinking:

Reply:
Thanks for the review. :)

From a functional point of view, I don't know if it should be displayed or not if empty.

Previously, from a technical point of view, an empty string was still displayed.
So I kept the same behavior. :sweat_smile: 

Not sure who I can contact about that.

@lordkrandel maybe

Reply:
Thanks for checking. :+1: 

I've updated the fix.

Reply:
Thanks ! Sending it for r+